.github/workflows/test.yml

```diff
@@ -104,6 +104,18 @@ jobs:
             -d postgres:${{ matrix.postgres-version }} \
             -c wal_level=logical
 
+      - name: Start PostgreSQL (Storage)
+        run: |
+          docker run \
+            --health-cmd pg_isready \
+            --health-interval 10s \
+            --health-timeout 5s \
+            --health-retries 5 \
+            -e POSTGRES_PASSWORD=postgres \
+            -e POSTGRES_DB=powersync_storage_test \
+            -p 5431:5432 \
+            -d postgres:${{ matrix.postgres-version }}
```

Comment:
Is it feasible to use a single Postgres server for both? (Not important if that's too much effort)

Reply:
I tested with a single server and that resulted in an endless "loop" of processing keep_alive operations. I think we might be able to work around that specific issue by potentially throttling keep_alive events, but that seems a bit hacky and might have other consequences that I'm not aware of. I'd like to address after the initial release if possible.